src/components/views/ViewsListTable.tsx

```diff
@@ -32,6 +38,25 @@ const ViewsListTable: React.FunctionComponent = () => {
             flex: 1,
             headerName: intl.formatMessage({ id: 'pages.people.views.viewsList.columns.owner' }),
         },
+        {
+            field: 'menu',
+            headerName: ' ',
+            renderCell: (props) => {
+                return (
+                    <ZetkinEllipsisMenu
+                        items={ [{
+                            id: 'delete-view',
+                            label: intl.formatMessage({
+                                id: 'pages.people.views.layout.ellipsisMenu.delete',
+                            }),
+                            onSelect: () => setSelectedViewToDelete(props.id as number),
```

Comment:
This had me stumped at first. I don't think we need to change it now, but I do think it would be cleaner with a function that triggers a confirm dialog, and calls back after user confirms, essentially the thing that @lowlandjuju has brewing in their PR.

But let's merge this and then figure out how to align the two. ✨